tst/EnergyPlus/unit/DataHeatBalance.unit.cc

```diff
@@ -985,3 +985,50 @@ TEST_F(EnergyPlusFixture, DataHeatBalance_setNodeSourceAndUserTemp)
     EXPECT_EQ(expectedNodeNumberAtSource, thisConstruct.NodeSource);
     EXPECT_EQ(expectedNodeNumberAtUserSpecifiedLocation, thisConstruct.NodeUserTemp);
 }
+
+TEST_F(EnergyPlusFixture, DataHeatBalance_AssignReverseConstructionNumberTest)
+{
+    int ConstrNum;
+    int expectedResultRevConstrNum;
+    int functionResultRevConstrNum;
+    bool ErrorsFound = false;
+
+    state->dataHeatBal->TotConstructs = 2;
+    state->dataConstruction->Construct.allocate(state->dataHeatBal->TotConstructs);
+    state->dataConstruction->LayerPoint.allocate(Construction::MaxLayersInConstruct);
+
+    auto &thisConstruct(state->dataConstruction->Construct(1));
+    auto &otherConstruct(state->dataConstruction->Construct(2));
+
+    // Test: For an interzone surface that uses the zone as the other side, a new
+    //       interzone surface for that zone has to be created.  That interzone has
+    //       to have the reversed construction.  If that reversed construction is
+    //       not used for any defined surface, the variable IsUsed is false.  However,
+    //       since it now is being used, IsUsed has to be set to true.  Prior to
+    //       work on Defect #8919, when using CondFD and a zone other side condition
+    //       for an interzone surface (implied or E+ internally created other side)
+    //       for a construction that was reversed in the input file but not used
+    //       for other defined surfaces, IsUsed was set to false and this construction
+    //       was then skipped in the CondFD routine that calculates the number of
+    //       nodes.  This led to a hard crash.  This test simply makes sure that
+    //       IsUsed is set to true when the reversed construction already exists.
```

Comment:
This helps me understand not only the fix you made, but also the intent of the unit test.  This is good stuff.  Thanks @RKStrand